### PR TITLE
Add request localization settings to the configured one

### LINKED ISSUE #14834: language switcher in admin not working
With the recent code the language switcher in the admin is not working. It stays on the same language.

image

@hishamco @MikeAlhayek

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #14834

### CODE REVIEW

**Comment by hishamco** [comment]
@giannik you could test this PR to ensure it resolve the issue

**Comment by giannik** [comment]
@hishamco yes this fixes it. 
thanks

File: src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs [hunk 1]
```diff
@@ -50,14 +50,14 @@ public override async ValueTask ConfigureAsync(IApplicationBuilder app, IEndpoin
...
-                options.SetDefaultCulture(defaultCulture);
-                options.AddSupportedCultures(supportedCultures);
-                options.AddSupportedUICultures(supportedCultures);
-            });
+            localizationOptions.CultureInfoUseUserOverride = !cultureOptions.IgnoreSystemSettings;
+
+            app.UseRequestLocalization(localizationOptions
+                .SetDefaultCulture(defaultCulture)
+                .AddSupportedCultures(supportedCultures)
+                .AddSupportedUICultures(supportedCultures));
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs:60]: I would first complete the `localizationOptions` configuration outside just before and then call `app.UseRequestLocalization(localizationOptions)` like in the setup startup and I think how it was originally done.

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs:60]: I think it's same because it call the chaining methods then return the object

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs:60]: IF think it's the same. It's just uglier (in this PR).

**Review by sebastienros — APPROVED** [review]

**Comment by sebastienros** [comment]
Bonus point if you can add a functional test.

**Review by Piedone — APPROVED** [review]

**Comment by Piedone** [comment]
That's more likely due to the various navbar changes: https://docs.orchardcore.net/en/latest/docs/releases/1.8.0/#navbar

**Pull request merged**